daml-lf/data/src/test/scala/com/digitalasset/daml/lf/data/TryOpsTest.scala

```diff
@@ -1,87 +0,0 @@
-// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-package com.daml.lf.data
-
-import org.scalatest.matchers.should.Matchers
-import org.scalatest.wordspec.AnyWordSpec
-import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
-import com.daml.lf.data.TryOps.Bracket.bracket
-
-import scala.util.{Failure, Success, Try}
-
-class TryOpsTest extends AnyWordSpec with Matchers with ScalaCheckDrivenPropertyChecks {
```

Comment:
Alternatively, you could deprecate this class.

Reply:
:+1: